src/components/DeploymentListItem.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <StateListItem class="deployment-list-item" :tags="tags" :state-type="deploymentState">
+    <template #name>
+      <p-link :to="routes.deployment(deployment.id)">
+        <p-heading :heading="5">
+          {{ deployment.name }}
+        </p-heading>
+      </p-link>
+    </template>
+
+    <template #relationships>
+      <div class="deployment-list-item__relationships-container">
+        <ListItemMetaFlowRun :title="localization.info.lastRun" :flow-run="lastRun" class="deployment-list-item__last-run" />
+        <ListItemMetaFlowRun :title="localization.info.nextRun" :flow-run="nextRun" class="deployment-list-item__next-run" />
+      </div>
+    </template>
+
+    <template v-if="deployment.workPoolName || deployment.workQueueName" #meta>
+      <template v-if="deployment.workPoolName">
+        <ListItemMeta :title="localization.info.workPool">
+          <WorkPoolIconText :work-pool-name="deployment.workPoolName" />
+        </ListItemMeta>
+      </template>
+
+      <template v-else-if="deployment.workQueueName">
+        <ListItemMeta :title="localization.info.workQueue">
+          <WorkQueueIconText :work-queue-name="deployment.workQueueName" />
+        </ListItemMeta>
+      </template>
+    </template>
+
+    <template #action>
+      <DeploymentMenu size="xs" :deployment="deployment" show-all @delete="handleDelete" />
+    </template>
+
+    <div class="deployment-list-item__schedule">
+      <DeploymentToggle :deployment="deployment" :disabled="!schedule" @update="handleUpdate" />
+
+      <div class="deployment-list-item__schedule-icon-text">
+        <p-icon icon="ClockIcon" />
+
+        <template v-if="schedule">
+          <div class="deployment-list-item__schedule-text">
+            <span :title="schedule?.toString({ verbose: true })">{{ scheduleText }}</span>
```

Comment:
is the `?` necessary since there's a `v-if="schedule"` on line 42? Is that just volar being dumb?

Reply:
Also, we have a `p-tooltip` component now. Which would be a better ux than the `title` attribute IMO.

Reply:
Ah! That was directly from the previous times we've used it, it looks like volar has fixed that!

Agreed, I forgot we had that, will update